### DISCUSSION TITLE
Webhooks calling the callback url infinite times if a Content record is updated.

### DISCUSSION THREAD

**Discussion by mdhameed** [strapi/strapi#7847] [Legacy Topics]
Webhooks calling the callback url infinite times if a Content record is updated. Adding of new record, it is fine it triggers the url only once but if we configure webhook for Edit then if a record is updated then the webhook triggering the event infinite times. Can anybody please check this issue and suggest a fix. Thanks.

**Comment by derrickmehaffy** [comment]
Can you please open up a bug report, following the bug report template? Thank you :hugs:

**Reply by mdhameed** [reply]
My apologizes. We found it to be issue in our app but not in webhooks, so we have to handle this and break the circular calls to strapi via graphQL.

**Reply by derrickmehaffy** [reply]
Gotcha cool :)
